### ISSUE TITLE
do you guyz have plan to make a tanoalife mission (sqm) file? 

### ISSUE THREAD

**Issue by HyunsukBaek** [ArmaLife/Framework#891]
im waiting !

**Comment by Earl-of-Lemongrab** [comment]
A Tanoa build is being made over at https://github.com/AsYetUntitled/Framework . It's a work in progress but I think they have an official testing server up for it...someone else might know?

**Comment by danielstuart14** [comment]
^ that's not official. I'm on vacation, and in the next week I'll need to finish my college final project. So, only after that I'll be able to start working on it.

**Comment by Earl-of-Lemongrab** [comment]
Oh sorry, got confused as all of the contributors from here seen to be making it...

**Comment by Jawshy** [comment]
> im waiting !

 

> You know what they say. If you want something done, best off doing it yourself. Otherwise sit in line and twiddle your thumbs. 
~ [tkcjesse](https://github.com/ArmaLife/Framework/issues/486#issuecomment-233634218)

**Comment by tkcjesse** [comment]
YES! People are finally quoting me!

**Comment by dbenson24** [comment]
@danielstuart14 "that's not official"
I mean it's being contributed to by most of the contributors of this repo, seems pretty official to me

**Comment by TAWTonic** [comment]
If it is not pushed to this repo then it's not official, doesn't matter if the same people contribute to a different repo.

**Comment by setoy** [comment]
That other repo seems a lot more polished than this one, will the changes from the other repo eventually be merged into this one?
